packages/framework/tests/Unit/Model/Feed/FeedCronModuleTest.php

```diff
@@ -26,6 +28,17 @@ public function testSleepExactBetweenFeeds(): void
             ->disableOriginalConstructor()
             ->getMock();
 
+        $feedModuleRepositoryMock = $this->getMockBuilder(FeedModuleRepository::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getAllScheduledFeedModules', 'getFeedModuleByNameAndDomainId'])
```

Comment:
`setMethods` is deprecated, I think you should use `onlyMethods`, see https://github.com/sebastianbergmann/phpunit/pull/3687

Reply:
changed, but it should be unified through the codebase

Reply:
We will probably do that during a future upgrade of PHPUnit :slightly_smiling_face: